assets/app/scripts/controllers/create.js

```diff
@@ -13,11 +14,11 @@ angular.module('openshiftConsole')
     var openshiftTemplates;
 
     // Templates with the `instant-apps` tag.
-    $scope.instantApps;
+    // $scope.instantApps;
```

Comment:
The intent may have been to show the variable while leaving it undefined? I would probably do `$scope.instantApps = undefined;`